client/me/purchases/payment/edit-card-details/index.jsx

```diff
@@ -71,26 +62,21 @@ const EditCardDetails = React.createClass( {
 	 * @param fields
 	 */
 	mergeCard( card, fields: {} ) {
+		const { name } = card;
+
 		return extend( {}, fields, {
-			name: card.name
+			name
```

Comment:
That's fine but you didn't convince me that two lines of code is better than one :).